src/index.ts

```diff
@@ -65,5 +65,6 @@ function vscodeSettings(): UnibeautifyVSCodeSettings {
 }
 
 export interface UnibeautifyVSCodeSettings {
+  defaultConfig: string;
```

Comment:
`UnibeautifyVSCodeSettings` should match `contributes.configuration` from `package.json`: https://github.com/muuvmuuv/vscode/blob/11c22960694b22f7b73e936cc36d83ab1bd49b40/package.json#L104-L122

For example, `enable` property: https://github.com/muuvmuuv/vscode/blob/11c22960694b22f7b73e936cc36d83ab1bd49b40/package.json#L110-L111

```
                "unibeautify.enabled": {
                    "type": "boolean",
                    "default": true,
```

So `enable` is of type `boolean | true` which can be simplified to `boolean`.

Let's look at `defaultConfig`: https://github.com/muuvmuuv/vscode/blob/11c22960694b22f7b73e936cc36d83ab1bd49b40/package.json#L115-L116

```
                "unibeautify.defaultConfig": {
                    "type": "string",
                    "default": null,
```

So `defaultConfig` is of type `string | null` not `string` (as it is in the interface at line 68).

It should look more like:

```
 export interface UnibeautifyVSCodeSettings {
-  defaultConfig: string;
+  defaultConfig: string | null;
```

😃 .

Also, I think `default: null` should actually be `default: undefined`. Since `undefined` is the value when it is not set, not `null`, if I am not mistaken?

@muuvmuuv please let me know if you have any questions. This PR is really close, there always seems to be *one more thing* 😝 .

Reply:
Okay, sorry I‘m not so good with typescript. Still learning, thanks!

Reply:
in package.json is only null valid, I have to wrap undefined in quotes to get it working. Are you ok with that?
Also I changed `string` to `string | undefined` inside the interface

Reply:
> I have to wrap undefined in quotes to get it working

Good to know. Then let's use `null`. `"undefined"` (string) is not what I want, if we can't have `undefined` then `null` will do 👍 . Use `defaultConfig: string | null`.

> Okay, sorry I‘m not so good with typescript. Still learning, thanks!

No worries! You're doing great! There's a lot going on here, with VSCode Extension's meta data in `package.json`'s `contributes.` and then keeping in sync with TypeScript 🎉 .